.github/mergify.yml

```diff
@@ -118,6 +118,40 @@ pull_request_rules:
   actions:
     merge:
       method: merge
+      # The dict of GitHub user ID and sign-off was built from
+      # https://github.com/instructlab/community/blob/main/MAINTAINERS.md
+      # Then, the sign-off was looked up in the git history
+      commit_message_template: |
+        {{ title }} (#{{ number }})
+
+        {{ body }}
+
+        {% set approved_by = {
```

Comment:
> my only concern is that this list will need to kept up-to-date with the Maintainers no? Should we add a note in the community repo that this needs to be updated when a Maintainer is added?

I think this list is not sustainable. If we must include approved by in the commit message, then let's just use the GitHub id rather than maintaining this new maintainers list.

Reply:
Now I have built the list and it defaults to the GitHub user ID if not present, so why not use it? We don't add maintainers frequently, don't you think saying it's "not sustainable" is a bit exaggerated? Thanks!

Reply:
It is a repetition of the source of truth and will deviate over time since attention won't be paid. It is also hard to copy this file to another instruct lab repo if they want to use mergify. 

If this goes in, I would appreciate removing me from the list.

Reply:
Sounds good. I think we've debated this enough, and it's time for others to weigh in. If the majority approves, I'll remove your sign-off from `.github/mergify.yml`. Thanks.